src/datasources/db/v2/postgresql-logger.service.ts

```diff
@@ -0,0 +1,89 @@
+import type { ILoggingService } from '@/logging/logging.interface';
+import { Injectable } from '@nestjs/common';
+import {
+  AbstractLogger,
+  type LoggerOptions,
+  type LogLevel,
+  type LogMessage,
+} from 'typeorm';
+
+@Injectable()
+export class PostgresqlLogger extends AbstractLogger {
+  public constructor(
+    public options?: LoggerOptions,
+    private readonly loggingService?: ILoggingService,
```

Comment:
We could inject the `loggingService` and not need pass it here.

Reply:
Not sure if I follow but `postgresConfig` has access to the `LoggingService` in the [relative factory](https://github.com/safe-global/safe-client-gateway/blob/ormMigrationAndLogger/src/app.module.ts#L140), and is passed there.

Reply:
in `package.json` on line 26 we are passing `orm.js` directly to `TypeOrm`
`"typeorm": "node --env-file=.env -r tsconfig-paths/register ./node_modules/typeorm/cli -d ./dist/src/config/entities/orm.config.js",`
for that reason it's not possible.

Reply:
Thank you for the clarification. Do we need a `logger` when running the command? We have a bash-specific configuration without one. Not sure if the [other change](https://github.com/safe-global/safe-client-gateway/pull/1988/files#r1803242982) has any influence either?

If it makes things too complicated though, let's keep it as is.

Reply:
We are not registering the logger when running the command.

Reply:
I see. I think this is now [resolved](https://github.com/safe-global/safe-client-gateway/pull/1988/files#r1803242982) then. Apologies for any confusion.